frontend/src/citizen-frontend/messages/ThreadView.tsx

```diff
@@ -171,13 +173,24 @@ export default React.memo(function ThreadView({
   )
   return (
     <ThreadContainer>
+      {title && type && (
+        <StickyTitleRow>
+          <H2 data-qa="thread-reader-title">{title}</H2>
+          <MessageTypeChip type={type} labels={i18n.messages.types} />
+        </StickyTitleRow>
+      )}
       {messages.map((message, idx) => (
-        <SingleMessage
-          key={message.id}
-          message={message}
-          title={idx === 0 ? title : undefined}
-          type={idx === 0 ? type : undefined}
-        />
+        <>
+          {idx === messages.length - 1 && !replyEditorVisible && (
+            <div style={{ position: 'relative' }}>
+              <span
+                style={{ position: 'absolute', top: '-100px' }}
```

Comment:
Instead of using a fixed amount of `100px` you should calculate the offset from `<StickyTitleRow>`'s box height. Also, consider using styled-components for consistency.

Reply:
Thanks, will do.